<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierEtudient2</class>
 <widget class="QDialog" name="ModifierEtudient2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modifier etudient</string>
  </property>
  <widget class="QPushButton" name="ptSave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMassar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtChamp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>massar :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>champ :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ancien valeur:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nouveux valeur :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
